src/main/java/org/killbill/billing/plugin/avatax/api/AvaTaxTaxCalculator.java

```diff
@@ -226,7 +227,7 @@ private GetTaxRequest toTaxRequest(final String companyCode,
         taxRequest.ExemptionNo = null;
         taxRequest.Discount = BigDecimal.ZERO;
         // Required for VAT
-        taxRequest.BusinessIdentificationNo = null;
+        taxRequest.BusinessIdentificationNo = PluginProperties.findPluginPropertyValue(BUSINESS_IDENTIFICATION_NUMBER, pluginProperties);
```

Comment:
Could you detail how you run the tests? 😅

With Avalara, you have to get a full account to make EU calculations, so it won't be easy to validate for real. However, indeed we could test that the request is still going through.